### ISSUE TITLE
Can not connect to mysql inside docker container

### ISSUE THREAD

**Issue by Abhinav1217** [sqlectron/sqlectron-gui#461]
When I tried to connect to mysql inside a docker container, It gives following error
```
ER_NOT_SUPPORTED_AUTH_MODE: Client does not support authentication protocol requested by server; consider upgrading MySQL client
```

I am new to docker and I have followes [this medium link](https://medium.com/coderscorner/connecting-to-mysql-through-docker-997aa2c090cc) to create mysql docker. I have tried testing connection using basic php db test script `mysqli_connect` using host PHP-FPM to mysql inside docker container, and it does work, But I can not connect to it using sqlectron. What I am missing.

**Comment by barneydmedia** [comment]
This is actually a bug with this projects mysql driver. We're waiting on the package to be updated and unfortunately can't do much about it until it is, although it is being worked on. See #442 for more info.

**Comment by Abhinav1217** [comment]
Good to know. Yes I found out that [mysql changed how they authenticate the connection. ](https://mysqlserverteam.com/mysql-8-0-4-new-default-authentication-plugin-caching_sha2_password/). Apparently this caused an issue discussion everywhere from Laravel to PDO to even JDBC. 

Simple solution is to not upgrade the mysql. Other solution is to execute following command in the mysql shell.
```
alter user 'username'@'localhost' identified with mysql_native_password by 'password';
```

I used the latter and was able to connect using sqlectron, along with other drivers within the app.
